testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/security/callerprincipal/jboss-ejb3.xml

```diff
@@ -21,16 +21,16 @@
   ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
   -->
 
-<jboss:jboss
-        xmlns="http://java.sun.com/xml/ns/javaee"
-        xmlns:jboss="http://www.jboss.com/xml/ns/javaee"
+<jboss:ejb-jar
+        xmlns="https://jakarta.ee/xml/ns/jakartaee"
+        xmlns:jboss="urn:jboss:jakartaee:1.0"
         xmlns:s="urn:security:1.1"
```

Comment:
should this be updated to urn:security:2.0 to match the new jboss-ejb-security_2_0.xsd?